core/src/testSubjects/java/junit/JunitTests.java

```diff
@@ -6,4 +6,10 @@ public class JunitTests {
 
   @Test
   public void test1() {}
+
+  public static class A {
+
+    @Test
```

Comment:
Wow, will JUnit actually run this method?  I had no idea 🙂

Reply:
I think it does. But, if it doesn't, I think that may be a different issue :). In fact, I'm unsure which version of JUnit is being supported here.

Reply:
I think it depends a bit on the JUnit version:

https://stackoverflow.com/questions/8758294/test-cases-in-inner-classes-with-junit

But in this case, since `A` is `static`, it will run the test method.